README.md

```diff
@@ -45,3 +45,5 @@ S2 Critical | Broken feature, workaround too complex & unacceptable
 S3 Major    | Broken feature, workaround acceptable
 S4 Low      | Functionality inconvenience or cosmetic issue
 
+## IDE specific settings
```

Comment:
I would put it under Development section rather than create a new one, but it doesn't really matter.